### DISCUSSION TITLE
Practica Pagina HTML

### DISCUSSION THREAD

**Discussion by AzaelBarbosa** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#207] [Show and tell]
Hola explores, les dejo mi practica aun no la finalizo, acabo de terminar el inicio y el diseño de la pagina, para manana la veran actualizada :), por lo pronto les dejo el inicio, ando desempolvando los conocimientos hace mucho que no programo en HTML y CSS ya que ando enfocado en .NET.

No es de pasteleria, la hice sobre videojuegos :)

Edit1: Ya esta actualizada :)

https://sheikahbrothers.web.app/index.html

**Comment by JuanSalvador73** [comment]
Me agrada. Si veo que será de videojuegos y no de pasteles :X

**Reply by AzaelBarbosa** [reply]
Gracias :D, ya tengo el Wireframe para terminarlo mañana, en el live del lunes el Commander dijo que no era necesario que fuera de pasteles

**Reply by LuceroLuciano** [reply]
Wow es de videojuegos, creí que era de películas ahaha.
Te quedo cool! 👌

**Comment by hackit92** [comment]
No sabía que se podía cambiar la temática, pero me gusto mucho como rescataste el estilo clásico boxed de los clásicos blogs de videojuegos, ya desde ahí se siente el ambiente gaming, felicidades!

**Comment by JonathanGrimaldo** [comment]
Me gusto se ve muy padre ese estilo, me recordó a los blogs clásicos 💯
